lib/classes/Service.js

```diff
@@ -211,15 +211,15 @@ class Service {
     });
   }
 
-  validate() {
+  async validate() {
```

Comment:
For future reference, I've just noticed that this change will also require an update here: https://github.com/serverless/serverless/blob/0f7b31b593595c49543eb40a9702180d14d3c4c9/lib/plugins/interactiveCli/initializeService.js#L96

Reply:
thanks, it seems like you did this already as a part of recent PR